From weiyi.huang@gmail.com  Thu Jul  2 13:05:52 2009
From: Huang Weiyi <weiyi.huang@gmail.com>
Date: Sat, 20 Jun 2009 19:16:51 +0800
Subject: Staging: cpc-usb: remove unused #include <linux/version.h>
To: gregkh@suse.de
Cc: devel@driverdev.osuosl.org, Huang Weiyi <weiyi.huang@gmail.com>
Message-ID: <1245496611-2464-1-git-send-email-weiyi.huang@gmail.com>


Remove unused #include <linux/version.h>'s in
drivers/staging/cpc-usb/cpc-usb_drv.c.

Signed-off-by: Huang Weiyi <weiyi.huang@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/cpc-usb/cpc-usb_drv.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/staging/cpc-usb/cpc-usb_drv.c
+++ b/drivers/staging/cpc-usb/cpc-usb_drv.c
@@ -28,7 +28,6 @@
 #include <asm/uaccess.h>
 #include <linux/usb.h>
 
-#include <linux/version.h>
 
 #include <linux/proc_fs.h>
 
